internal/api/connectors/sources/sources_test.go

```diff
@@ -0,0 +1,71 @@
+package sources
+
+import (
+	"playbook-dispatcher/internal/common/config"
+	"playbook-dispatcher/internal/common/utils"
+	"playbook-dispatcher/internal/common/utils/test"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/zap"
+)
+
+var _ = Describe("Sources", func() {
+	Describe("GetSources", func() {
+		It("interperates response correctly", func() {
+			doer := test.MockHttpClient(200, `{"data": [{"id": "1", "name": "test", "availability_status": "connected"}]}`)
+
+			client := NewSourcesClientWithHttpRequestDoer(config.Get(), &doer)
+			ctx := utils.SetLog(test.TestContext(), zap.NewNop().Sugar())
```

Comment:
I think this line can be replaced with the following:
```suggestion
			ctx := test.TestContext()
```

Same thing in the rest of the tests (line 36, 49, 64)